src/pods/configure-verbs/configure-verbs.container.tsx

```diff
@@ -27,6 +27,7 @@ export const ConfigureVerbsContainer = () => {
       []
     );
 
+    localStorage.setItem('selectedVerbs', JSON.stringify(selectionKeys));
```

Comment:
Let's add a wrapper for the localStorage operations, this wrapper:
- Will take care of error handling (e.g. some broswsers in incognito mode can have disallowed accesing local storage).

- Will take care of stringyfing the values

About the Key 'selectedVerbs' we should store this in a const